packages/fxa-settings/src/pages/SigninConfirmed/index.tsx

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { RouteComponentProps } from '@reach/router';
+import Ready from '../../components/Ready';
+
+type SigninConfirmedProps = {
+  continueHandler?: Function;
+  serviceName?: string;
+};
+
+const SigninConfirmed = ({
```

Comment:
I don't suspect this is a relevant or useful comment, but something my eyes spotted after reviewing a different PR&hellip; sometimes we use "Signin" and other times we use "SignIn". Again, not sure it's relevant versus possibly accounts vs subplat, or whatever.

This silly grep should return all instances of "signIn" or "SignIn" ("s" or "S", but capital "I"), that aren't tests or Storybook stories, in .js or .ts or .tsx (or even rogue .jsx) files:
```sh
git grep -E "\b[sS]ign[I]n.*\b" packages | grep -E "\.([jt]sx?):" | grep -Ev "\.(spec|test|stories)\."
```

Reply:
Oooh that's interesting

Reply:
I'm fine with either but I hope we can be consistent heh 🫣